### DISCUSSION TITLE
Whats the difference betweeen initial_royalties and royalties in the tenk contract ?

### DISCUSSION THREAD

**Discussion by akshay-anv** [TENK-DAO/tenk#50] [Q&A]
As the titles suggests , this is an amazing repo and frankly speaking I dont know why its even public though compliments aside whats the difference between initial_royalties  and  royalties  , what do those numbers mean ?

**Comment by willemneal** [comment]
Royalties are the split of the money spent on minting (`initial_royalties`) or reselling (`royalties`).  The units are in [basis points](https://en.wikipedia.org/wiki/Basis_point), which treat 100% as 10,000.  This lets you use integers to represent fractions of a percent.  E.g. 7.75% = 775 basis points.  For royalties you say what percent of the total is to be split and then have a mapping for each of the accounts that split the funds.

And this repo is open so that others can learn and help it grow.  And PRs are welcome!
